plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/general/SwitchGroup/SwitchGroupItem.tsx

```diff
@@ -0,0 +1,144 @@
+import { Icon } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+
+import { groupItem } from "../../types";
+
+type Props = {
+  items?: groupItem[];
+  switchGroups: groupItem[];
+  handleMoveDown: (idx: number) => void;
+  handleMoveUp: (idx: number) => void;
+  handleRemove: (idx: number) => void;
+  handleModifyGroup: (item: string, indx: number) => void;
+  handleModifyGroupTitle: (title: string, indx: number) => void;
+};
+
+const SwitchGroupItems: React.FC<Props> = ({
+  items,
+  switchGroups,
+  handleMoveDown,
+  handleMoveUp,
+  handleRemove,
+  handleModifyGroup,
+  handleModifyGroupTitle,
+}) => {
+  return (
+    <Wrapper>
+      {items?.map((item, idx) => (
+        <Item key={idx}>
+          <ItemControls>
+            <Icon icon="arrowUpThin" size={16} onClick={() => handleMoveUp(idx)} />
+            <Icon icon="arrowDownThin" size={16} onClick={() => handleMoveDown(idx)} />
+            <Icon icon="trash" size={16} onClick={() => handleRemove(idx)} />
+          </ItemControls>
+          <Field>
+            <FieldTitle>グループ</FieldTitle>
+            <FieldValue>
+              <SelectWrapper
+                value={item.group}
+                onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
+                  handleModifyGroup(e.target.value, idx);
+                }}>
+                <option selected disabled defaultValue={""} />
```

Comment:
I think the first group in the switchGroups array should be the default selected option.